src/main/CommandLine.cpp

```diff
@@ -554,6 +563,13 @@ runCatchup(CommandLineArgs const& args)
                 config.DISABLE_XDR_FSYNC = true;
             }
 
+            if (!stream.empty())
+            {
+                // Allow overriding config METADATA_OUTPUT_STREAM using
+                // command line --metadata-output-stream
+                config.METADATA_OUTPUT_STREAM = stream;
```

Comment:
Should it just blow up if the setting was already in the config file? I kinda hate the notion of "override" that just creates ambiguity.

Reply:
Fair, pushed variant that's exclusive.